requirements/requirements_asr.txt

```diff
@@ -16,6 +18,7 @@ pyannote.metrics
 pydub
 pyloudnorm
 resampy
+riva-asrlib-decoder
```

Comment:
Please, make these dependencies optional
I suggest adding installation to `scripts/asr_language_modeling/ngram_lm/install_beamsearch_decoders.sh`